docs/code_ref/roi.rst

```diff
@@ -1,7 +1,10 @@
 SunPy roi
 =========
 
+.. warning::
+
+   The roi module is still in development, the API is subject to change.
```

Comment:
I mean techncially true, but we have levels of stability, and roi is at the bottom.